### DISCUSSION TITLE
x86 openwrt下启动不了，最新版本

### DISCUSSION THREAD

**Discussion by AspirantH** [AlistGo/alist#5156] [General]
![1](https://github.com/alist-org/alist/assets/93640658/60e06d00-0d54-4684-8edb-98957af53e86)
![2](https://github.com/alist-org/alist/assets/93640658/1a7a9207-1efe-4f3c-90f8-b62f7d267133)

**Comment by AspirantH** [comment]
版本https://github.com/sbwml/luci-app-alist
日志：not enbale search

**Comment by Troray** [comment]
重启一次设备再尝试

**Reply by AspirantH** [reply]
重启过了，openwrt启动项里的alist也点过启动了

**Comment by AspirantH** [comment]
有没有好心人帮忙看看，https://post.smzdm.com/p/akk3d6q9/
这里windows版本有提到同样的提示not enable search
